vsphere/internal/virtualdevice/virtual_machine_cdrom_subresource.go

```diff
@@ -378,6 +384,25 @@ func CdromPostCloneOperation(d *schema.ResourceData, c *govmomi.Client, l object
 	return l, spec, nil
 }
 
+// ValidateCdromConfig takes a CdromSubresource and ensures that the associated configuration is valid
+func (r *CdromSubresource) ValidateCdromConfig() error {
+	log.Printf("[DEBUG] ValidateCdromConfig: Beginning CDROM configuration validation")
```

Comment:
Subresource debug lines should include the specific resource address. For this log entry, this means it would look like:

```
log.Printf("[DEBUG] %s: Beginning CDROM configuration validation", r)
```

Function does not necessarily need to be referenced here.